core/domain/prod_validators.py

```diff
@@ -3153,4 +2811,4 @@ def _get_external_id_relationships(cls, item):
                 user_models.UserSettingsModel,
                 [item.user_id]
             )
-        ]
+        ]
```

Comment:
Missing newline at the end... How did linters not pick this up?

Reply:
I have made the changes.